elyra/pipeline/processor_kfp.py

```diff
@@ -68,10 +69,14 @@ def process(self, pipeline):
 
             self.log.debug("Creating temp directory %s", temp_dir)
 
+            engine = runtime_configuration.metadata.get('engine')
             # Compile the new pipeline
             try:
                 pipeline_function = lambda: self._cc_pipeline(pipeline, pipeline_name)  # nopep8 E731
-                kfp.compiler.Compiler().compile(pipeline_function, pipeline_path)
+                if 'Argo' == engine:
+                    kfp.compiler.Compiler().compile(pipeline_function, pipeline_path)
```

Comment:
Would be great if we could log which compiler was used. This way there's no ambiguity. (Same comment below)